src/Columns/Column.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+
+namespace Backpack\CRUD\Columns;
+
+use Backpack\CRUD\Facades\CRUDPanel;
+
+abstract class Column
+{
+    protected $type;
+    protected $data = [];
+
+    /**
+     * Column constructor.
+     * @param null $data
+     */
+    function __construct($data = null)
+    {
+        if (is_array($data)) {
+            $this->data = $data;
+        }
```

Comment:
You know it would be nice if this also had a:

```
else if (isset($data)) {
  throw new \Exception('What is that?');
}
```

OR

```
function __construct($data = []) {
}
```